build/csv/ceph/rook-ceph-operator.clusterserviceversion.yaml

```
@@ -1863,7 +1863,7 @@ metadata:
   name: rook-ceph-operator.v4.15.0
   namespace: placeholder
   relatedImages:
-    - image: docker.io/rook/ceph:master
+    - image: docker.io/rook/ceph:v1.13.0.399.g9c0d795e2
```

```suggestion
    - image: rook/ceph:v1.13.0.399.g9c0d795e2
```

seems like this also needs to be updated?

---

This is the result of above update. This keeps the container image and relatedImages field same in the csv

---

@Nikhil-Ladha I see this in related images `image: rook/ceph:master` which is not same as above.

---

Yeah, that's the placeholder.
The actual value is updated via the `gen-csv` file. over here: https://github.com/red-hat-storage/rook/blob/master/build/csv/csv-gen.sh#L53